README.md

```diff
@@ -90,3 +90,15 @@ To make it permanent add this line to your `~/.zshrc`.
 ### Documentation
  - [OpenCompose file reference documentation](https://github.com/redhat-developer/opencompose/blob/master/docs/file-reference.md)
  - You can find more in [doc/](https://github.com/redhat-developer/opencompose/tree/master/docs) folder
+
+### Contact
+We always welcome your feedback and thoughts on the project! Come and join us on [Slack](https://opencompose.slack.com) and join our [mailing list](https://groups.google.com/forum/#!forum/opencompose).
+
+#### Slack
+[opencompose.slack.com](https://opencompose.slack.com)
```

Comment:
I think we should have both. There should be a channel for discussing the tool, helping users, ... And we should also talk on #sig-apps when we want to engage Kubernetes community; on topics that relate to them. I don't think that spamming #sig-apps with discussing bugs, technological discussions and others would do any good.